sdk/monitor/monitor-opentelemetry-exporter/src/platform/nodejs/context/context.ts

```diff
@@ -94,19 +92,19 @@ export class Context {
         Context.appVersion[packageJsonPath] = packageJson.version;
       }
 
-      this.tags[this.keys.applicationVersion] = Context.appVersion[packageJsonPath];
+      this.tags["ai.application.ver"] = Context.appVersion[packageJsonPath];
     }
   }
 
   private _loadDeviceContext(): void {
-    this.tags[this.keys.deviceId] = "";
-    this.tags[this.keys.cloudRoleInstance] = os && os.hostname();
-    this.tags[this.keys.deviceOSVersion] = os && `${os.type()} ${os.release()}`;
-    this.tags[this.keys.cloudRole] = Context.DefaultRoleName;
+    this.tags["ai.device.id"] = "";
+    this.tags["ai.cloud.role"] = Context.DefaultRoleName;
+    this.tags["ai.cloud.roleInstance"] = os && os.hostname();
+    this.tags["ai.device.osVersion"] = os && `${os.type()} ${os.release()}`;
 
     // not yet supported tags
-    this.tags["ai.device.osArchitecture"] = os && os.arch();
-    this.tags["ai.device.osPlatform"] = os && os.platform();
+    this.tags["ai.device.osArchitecture" as ContextTagKeys] = os && os.arch();
+    this.tags["ai.device.osPlatform" as ContextTagKeys] = os && os.platform();
```

Comment:
These aren't included in tags types, but have always been sent. I'll investigate if it makes sense to add them to the swagger contract